Packs/CTIX/Integrations/CTIX/CTIX_test.py

```diff
@@ -0,0 +1,141 @@
+import io
+import json
+
+'''CONSTANTS'''
+
+BASE_URL = "http://test.com/"
+ACCESS_ID = "access_id"
+SECRET_KEY = "secret_key"
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def test_ip(requests_mock):
+    from CTIX import Client, ip_details_command
```

Comment:
unit tests should contain a given-when-then section.
e.g.
Given
\- 6.7.8.9
When
\- collecting data for an existing indicator
Then
\- ensure the return data contains ...